### PR TITLE
Level magic numbers

### PR DESCRIPTION
**Pull request by oldergod**
## Overview

Magic numbers were used around level ids. I created 3 constants to be used instead as well as simplifying the way `Level` subclasses are instantiated.

### CODE REVIEW

File: app/src/test/java/io/github/droidkaigi/confsched2018/DummyDataCreator.kt [hunk 1]
```diff
@@ -77,8 +79,7 @@ fun createDummySessionFeedbackEntities(): List<SessionFeedbackEntity> {
 
 }
 
-fun createDummySessionFeedback(sessionId: String = DUMMY_SESSION_ID1,
-                               sessionTitle: String = DUMMY_SESSION_TITLE1): SessionFeedback {
```
Comment (oldergod) [inline app/src/test/java/io/github/droidkaigi/confsched2018/DummyDataCreator.kt]: 無用

**Comment by takahirom** [comment]
Thanks! 🔢 
It became easier to read!

**Comment by takahirom** [comment]
Please fix this 🙏 
e: /home/circleci/project/app/src/test/java/io/github/droidkaigi/confsched2018/presentation/sessions/feedback/SessionsFeedbackViewModelTest.kt: (47, 25): Too many arguments for public fun createDummySessionFeedback(sessionId: String = ...): SessionFeedback defined in io.github.droidkaigi.confsched2018 in file DummyDataCreator.kt

**Comment by takahirom** [comment]
LGTM 👍

**Comment by oldergod** [comment]
すみませんでした、対応しました。

**Pull request merged**